src/main/groovy/templates/tasks/RestProject.groovy

```diff
@@ -166,6 +166,11 @@ class RestProject {
                     resourceName: resourceName, servicePackage: "${servicePackage}"
         }
 
+        basicProject.applyTemplate("src/main/java/${servicePackagePath}/config") {
```

Comment:
i believe this is in the wrong place - this is something we'd want in createRestBase b/c we only want a single jersey config for a project rather than one per resource